pytest_plugins/fixture_markers.py

```diff
@@ -6,19 +7,42 @@
 
 def pytest_generate_tests(metafunc):
     if 'rhel_contenthost' in metafunc.fixturenames:
-        rhel_parameters = []
-        for ver in settings.content_host.rhel_versions.keys():
-            # prefer nick-specific deploy workflow before using the default one
-            workflow = settings.content_host.rhel_versions[ver].get(
-                'deploy_workflow', settings.content_host.deploy_workflow
+        function_marks = getattr(metafunc.function, 'pytestmark', [])
+        # process eventual rhel_version_list markers
+        matchers = [i.args for i in function_marks if i.name == 'rhel_ver_list']
+        list_params = []
+        for matcher in matchers:
+            list_params.extend(
+                [
+                    param
+                    for param in settings.content_host.rhel_versions.keys()
+                    if str(param) in str(matcher)
```

Comment:
This will ease the code undesranding a bit:
```suggestion
                    if str(setting_rhel_ver) in str(fn_rhel_ver)
```

Reply:
suggestion implemented